src/components/DevTools/data/cTypes.json

```diff
@@ -3,6 +3,7 @@
     "schema": {
       "$id": "DriversLicense",
```

Comment:
should be  - `kilt:ctype:0x5366521b1cf4497cfe5f17663a7387a87bb8f2c4295d7c40f3140e7ee6afc41b`